receiver/googlecloudspannerreceiver/internal/filter/itemcardinality.go

```diff
@@ -0,0 +1,184 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filter
+
+import (
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/ReneKroon/ttlcache/v2"
```

Comment:
How well maintained is the library? It looks like it forked an old Microsoft package that is now in RO.

Should this be a concern?

Reply:
I do not think this should be a concern. I watched on few of cache related libraries and only this one has realtime value expiration after TTL reached, all the rest perform real expiration of item only in case when performing some cache related operations, like get, etc. This also allows me to know accurate exact number of items cached(and not expired) at particular moment of time. https://github.com/ReneKroon/ttlcache - this is the URL of library - latest update was performed 16 days ago - so I do not see issues with maintanance. I case you have better candidate - I am open to discussion.